back/config/schemas/settings.schema.json.erb

```diff
@@ -746,10 +746,10 @@
         }
       },
 
-      "konveio_surveys": {
+      "konveio_document_annotation": {
```

Comment:
Based on [this comment](https://citizenlab.atlassian.net/browse/CL-3466?focusedCommentId=30155), should it be `"default": false` for both allowed and enabled?

Reply:
> Are there any active platforms with this feature configured? If so, are we going to migrate this value for them?

Very good point. I had forgotten to consider this. I have added a todo item to this PR description.

Reply:
> Based on [this comment](https://citizenlab.atlassian.net/browse/CL-3466?focusedCommentId=30155), should it be `"default": false` for both allowed and enabled?

Yes, probably. I left them set to `true` for local testing. It might be good to confirm with Irene exactly what is desired.